src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerListener.cs

```diff
@@ -10,23 +10,27 @@ namespace Microsoft.Azure.WebJobs.Extensions.DocumentDB
     using Microsoft.Azure.Documents;
     using Microsoft.Azure.Documents.ChangeFeedProcessor;
     using Microsoft.Azure.Documents.Client;
+    using Microsoft.Azure.WebJobs.Host;
     using Microsoft.Azure.WebJobs.Host.Executors;
     using Microsoft.Azure.WebJobs.Host.Listeners;
 
     internal class CosmosDBTriggerListener : IListener, IChangeFeedObserverFactory
     {
         private readonly ITriggeredFunctionExecutor executor;
         private readonly ChangeFeedEventHost host;
```

Comment:
Is `ChangeFeedEventHost` mockable in any way? It'd be nice to have some tests to make sure this works, but that may involve creating a wrapper around this (like I did in the other bindings for `DocumentClient`). Since the logic is fairly simple, it may not be worth the effort...

Reply:
In the new version of the Host we are working on, it is, we have a whole complete set of Unit tests on the library. This version however, is not completely mockable :(